src/components/PopUpRevisionContainer/PopUpRevisionContainer.js

```diff
@@ -87,15 +87,16 @@ const PopUpRevisionContainer = ({
 }
 
 /**
+ * Component which renders the PopupRevisionTimeline component, which displays a popup containing the revisions in a timeline form, which the user can filter on.
+ *
+ * @component
  *
- * @param {object} props
- * @param {boolean} open - Indicating if the timeline popup is open
- * @param {object} revisionListItems - List item children components
- * @param {object} setOpen - Function to toggle the timeline popup
- * @param {object} setRevisionOverviewOpen - Function to toggle the revision overview popup
- * @param {object} titleSingular - Singular type title of the object we are viewing
- * @param {object} revisionObjects - Array of revisions
- * @returns Component that renders a toggleable popup with a timeline of revisions
+ * @param {boolean} open - Parameter used to display the Transition component.
+ * @param {array} revisionListItems - Parameter containing a list of revision list items.
+ * @param {function} setOpen - Function to close the revision timeline popup.
+ * @param {function} setRevisionOverviewOpen - Function to edit parent state
+ * @param {object} titleSingular - Title of the object in a singular form
```

Comment:
string

Reply:
I've changed the parameter description and the type to string.